crates/l2/prover/bench/src/rpc/db.rs

```diff
@@ -505,14 +506,54 @@ impl LevmDatabase for RpcDB {
 /// calculate all posible child nodes.
 fn get_potential_child_nodes(proof: &[NodeRLP], key: &PathRLP) -> Option<Vec<Node>> {
     // TODO: Perhaps it's possible to calculate the child nodes instead of storing all possible ones.
-    let trie = Trie::from_nodes(
-        proof.first(),
-        &proof.iter().skip(1).cloned().collect::<Vec<_>>(),
-    )
-    .unwrap();
+    let proof_missing_key = match proof.split_first() {
```

Comment:
Can we not use from_nodes here as well?

Reply:
This was made before the `from_nodes` function accepted inconsistent tries. Should I change back both `get_potential_child_nodes` functions?